CHANGES.md

```diff
@@ -71,12 +71,14 @@ Highlights
 
 Configuration Changes
 ---------------------
-* made 'jemalloc' the default allocator for all platforms other than Cygwin
+* made 'jemalloc' the default allocator for most configurations
   (see http://chapel.cray.com/docs/1.13/usingchapel/chplenv.html#chpl-mem)
-* made 'cray-prgenv-cray' default to using qthreads+hwloc and 'ugni'
+* made 'cray-prgenv-cray' default to using 'qthreads+hwloc' and 'ugni'
   (see  http://chapel.cray.com/docs/1.13/platforms/cray.html)
 * added support for 're2' when using Cray and PGI compilers
 * changed 'knc' to default to using 'qthreads+hwloc'
+* deprecated 'knc' support
+  (see  http://chapel.cray.com/docs/1.13/platforms/knc.html)
```

Comment:
Thanks for adding this -- I thought about it but couldn't figure out where it should go.  This is a good place.